Extension/.vscode/launch.json

```diff
@@ -2,6 +2,12 @@
 {
 	"version": "0.1.0",
 	"configurations": [
+		{
+			"type": "node",
+			"request": "launch",
+			"name": "Launch Program",
```

Comment:
Can you pick a better "name"? 😃

Reply:
This will be removed. I was using this to temporary debug generateOptionsSchema